### PR TITLE
Chore: increase test coverage

### PR DESCRIPTION
**Pull request by AnayoOleru**
#### Description
This PR increase create article test coverage
#### Type of change

- [x] create article test

#### How Has This Been Tested?

- [x] Unit testing

#### Checklist:
- [x] create article test
#### PT-ID

N/A

#### Screenshots
N/A
#### Questions:
N/A

### CODE REVIEW

**Review by amaechi-chuks — APPROVED** [review]
Looks good

**Review by vic3king — CHANGES_REQUESTED** [review]
@AnayoOleru this is really nice +2.9% on our coverage. However could you please try testing you `actions` and `reducers` this would further increase our coverage?

**Review by sojida — COMMENTED** [review]
Great work on getting our test coverage up by 2.9%. Although, we believe there's more testing to do with your actions and reducers. Thanks

**Review by walsamlee — APPROVED** [review]

File: src/components/presentations/NewArticle/__test__/NewArticle.test.jsx [hunk 1]
```diff
@@ -0,0 +1,111 @@
...
+    const event = {
+      preventDefault: jest.fn(),
+    };
+
+    const component = shallow(<NewArticle {...props} />);
+    component.instance().saveOrPublish(event, { isDraft: false });
+    component.instance().setState({
+      ...article,
+    });
+    expect(component.state('title')).toEqual('Sciencetist dont die');
```
Comment (rotimi-babalola) [inline src/components/presentations/NewArticle/__test__/NewArticle.test.jsx]: `Sciencetist` should be `Scientist`and `dont` should be `don't`

**Review by Akinmyde — APPROVED** [review]

**Review by vic3king — APPROVED** [review]

**Pull request merged**